call_16_test.go

```diff
@@ -0,0 +1,27 @@
+//go:build !go_tarantool_call_17
+// +build !go_tarantool_call_17
+
+package tarantool_test
+
+import (
+	"testing"
+
+	. "github.com/tarantool/go-tarantool"
+)
+
+func TestConnection_Call(t *testing.T) {
+	var resp *Response
+	var err error
+
+	conn := connect(t, server, opts)
+	defer conn.Close()
```

Comment:
I think we need such tests for ConnectionPool and ConnectionMulti. Looks like we won't test ConnectionPool.Call() and ConnectionMulti.Call().

Reply:
Done.